### PR TITLE
Jenkins 47550 use credential store

### PR DESCRIPTION
**Pull request by mvk**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -85,74 +100,102 @@
...
+
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>animal-sniffer-maven-plugin</artifactId>
+                <version>${animal-sniffer-maven-plugin.version}</version>
+                <configuration>
+                    <signature>
+                        <groupId>org.codehaus.mojo.signature</groupId>
+                        <artifactId>java18</artifactId>
+                        <version>1.0</version>
```
Comment (mvk) [inline pom.xml]: @alitet01, maybe export this version into `<properties></properties>` section?

File: src/main/java/io/relution/jenkins/awssqs/SQSTriggerQueue.java [hunk 2]
```diff
@@ -76,14 +86,39 @@
...
-            final Secret secretKey,
+            final String credentialsId,
             final Integer waitTimeSeconds,
             final Integer maxNumberOfMessages) {
+
         this.uuid = StringUtils.isBlank(uuid) ? UUID.randomUUID().toString() : uuid;
-        this.accessKey = accessKey;
-        this.secretKey = secretKey;
         this.nameOrUrl = nameOrUrl;
+        this.credentialsId = credentialsId;
```
Comment (mvk) [inline src/main/java/io/relution/jenkins/awssqs/SQSTriggerQueue.java]: @alitet01 please fix indentation - if the file is using spaces, use spaces.

File: src/main/java/io/relution/jenkins/awssqs/SQSTriggerQueue.java [hunk 3]
```diff
@@ -298,30 +352,32 @@ public FormValidation doCheckNameOrUrl(@QueryParameter final String value) {
         }
 
         public FormValidation doCheckUuid(@QueryParameter final String value) {
-            if (StringUtils.isBlank(value)) {
```
Comment (mvk) [inline src/main/java/io/relution/jenkins/awssqs/SQSTriggerQueue.java]: level of nesting is growing here, unnecessarily, and this degrades readability.
Unless there is a logic missed with "linearization", why doing it ?

**Comment by alitet01** [comment]
'Improve readability of code'  commit makes requested changes

**Pull request merged**